From: Sven Roederer <devel-sven@geroedel.de>
Date: Thu, 8 Apr 2021 21:19:46 +0200
Subject: imagebuilder: unset BINARY_FOLDER and DOWNLOAD_FOLDER in result

Setting these folder during buildtime will embedd their location via the .config file.
This results in trying to use the same subfolders on the (other) machine running the
imagebuilder, where these might no accessible at all.

Signed-off-by: Sven Roederer <devel-sven@geroedel.de>

diff --git a/target/imagebuilder/Makefile b/target/imagebuilder/Makefile
index f9c08776a8a68a3b724f32059851e307cdc2a37a..86956ba7ffb6013cef82f7625782ad227015427b 100644
--- a/target/imagebuilder/Makefile
+++ b/target/imagebuilder/Makefile
@@ -29,6 +29,8 @@ $(BIN_DIR)/$(IB_NAME).tar.xz: clean
 	mkdir -p $(IB_KDIR) $(IB_LDIR) $(PKG_BUILD_DIR)/staging_dir/host/lib \
 		$(PKG_BUILD_DIR)/target $(PKG_BUILD_DIR)/scripts $(IB_DTSDIR)
 	-cp $(TOPDIR)/.config $(PKG_BUILD_DIR)/.config
+	$(SED) 's/^CONFIG_BINARY_FOLDER=.*/# CONFIG_BINARY_FOLDER was reset by Imagebuilder/' $(PKG_BUILD_DIR)/.config
+	$(SED) 's/^CONFIG_DOWNLOAD_FOLDER=.*/# CONFIG_DOWNLOAD_FOLDER was reset by Imagebuilder/' $(PKG_BUILD_DIR)/.config
 	$(CP) -L \
 		$(INCLUDE_DIR) $(SCRIPT_DIR) \
 		$(TOPDIR)/rules.mk \
